### DISCUSSION TITLE
openshift alternative

### DISCUSSION THREAD

**Discussion by jaqbtor** [SKCraft/Launcher#474] [I need help!] [answered]
Is there an alternative openshift somewhere, with a tutorial for launcher, because the site has changed a lot and i have a problem with creation PHP app.

**Comment by hedgehog1029** [comment, marked as answer]
We have a [list of recommended web hosts](https://github.com/SKCraft/Launcher/wiki/Website-Hosting) on the wiki.

**Reply by jaqbtor** [reply]
Thanks, I am blind
